src/xdocs/config_annotation.xml

```diff
@@ -1,1571 +1,94 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would personally rename this file to `xdocs/checks/annotation/index.html`. What do you think?

Reply:
I agree.

Reply:
Agreed.

Reply:
@rnveach , please look also.